drivers/user_directory_report/app/views/user_directory_report/warehouse_reports/users/report.haml

```diff
@@ -0,0 +1,27 @@
+- content_for :title, _('User Directory Report')
+
+= render 'warehouse_reports/breadcrumbs', tab: 'operational'
+
+.o-page
+  .o-page__header
+    - @prompt = 'Search Users'
+
+    .o-page__controls
+      .o-page__search
+        = render 'search_form', tooltip: 'Search on a user\'s first name, last name or email address', aria: { label: 'User Search' }, data: {placement: 'bottom'}
+
+.row
+  .col-sm-8
+    %h1= content_for :title
+
+- if cas_available?
+  %ul.nav.nav-tabs{role: :tablist}
+    %li.nav-item{role: :presentation}
+      %a{href: 'warehouse', class: nav_link_classes('warehouse', @user_source)}
```

Comment:
the should be `= link_to` instead of a manual href.  It probably doesn't matter much, but it will make maintenance easier later.

Reply:
✅